aspnetcore/fundamentals/native-aot.md

```diff
@@ -138,9 +138,13 @@ Because unused code is trimmed during publishing for native AOT, the application
 </Project>
 ```
 
-With the project file updated, run the `dotnet build` command (`publish` isn't necessary to view generated code). In the built output file there will be a `obj/Debug/net8.0/generated/` directory which contains all the generated files for the project.
+Run the `dotnet build` command. `publish` isn't necessary to view generated code. The built output contains an `obj/Debug/net8.0/generated/` directory and all the generated files for the project.
 
-When `dotnet publish` is run, the project source files and generated source files are compiled as normal and then outputted assemblies are passed into an native IL compiler which produces the native executable which contains the native machine code to run the application.
+When `dotnet publish` is run:
+
+* The project source files and generated source files are compiled.
+* Generated assemblies are passed to an native IL compiler which produces the native executable. The native executable contains the native machine code.
+* The native app is ready to run.
```

Comment:
```suggestion
The `dotnet publish` command:

* Compiles the source files.
* Generates files which are are compiled.
* Passes generated assemblies to an native IL compiler. The IL compiler produces the native executable. The native executable contains the native machine code.
```

```suggestion
When `dotnet publish` is run:

* The project source files and generated source files are compiled.
* Generated assemblies are passed to an native IL compiler which produces the native executable. The native executable contains the native machine code.
* The native app is ready to run.
```

Reply:
I prefer this, please accept this suggestion.

Reply:
I would conflate the first two bullet points since the source generated files and the user-provided source files are compiled at the same time. There isn't two different compiation steps to go from C# code to DLL. There is a second IL compilation step to go from DLL/IL to native executable.